src/conversion.jl

```diff
@@ -15,7 +15,7 @@ function convert{R, C, R2, C2, T}(MT::Type{Mat{R, C, T}}, b::Mat{R2, C2, T})
 end
 
 #conversion
-convert{T <: Tuple}(::Type{T}, x::Real)                   = ntuple(ConstFunctor(eltype(T)(x)), Val{length(T.parameters),})
+convert{N}(T::Type{NTuple{N}}, x::Real) = ntuple(ConstFunctor(eltype(T)(x)), Val{N})
```

Comment:
I remember that this was at a time the only not totally terrible work-around... :( But maybe this has changed. The problem was the ambiguity with the default constructor for single arguments (since e.g `Vec` has only one field). Not sure what the reason was, that it works for the other single argument constructors in `core.jl`...